go/libraries/doltcore/env/actions/branch.go

```diff
@@ -28,7 +28,7 @@ import (
 
 var ErrAlreadyExists = errors.New("already exists")
 var ErrCOBranchDelete = errors.New("attempted to delete checked out branch")
-var ErrUnmergedBranchDelete = errors.New("attempted to delete a branch that is not fully merged into master; use `-f` to force")
+var ErrUnmergedBranchDelete = errors.New("attempted to delete a branch that is not fully merged into main; use `-f` to force")
```

Comment:
This is a confusing error message for existing customers without a main branch.

I would just use "its parent" instead of naming the branch.